src/components/paragraphs/sum-at-a-glance/at-a-glance-paragraph.tsx

```diff
@@ -87,6 +74,19 @@ const SumAtAGlanceParagraph = ({paragraph, ...props}: Props) => {
             />
           )}
         </div>
+
+        <div
+          className={twMerge(
+            "mt-auto h-fit border-archway-dark border-opacity-50 p-20",
+            clsx({
+              "border-l-3": !headingOnLeft,
+              "border-r-3 @6xl:order-first": headingOnLeft,
+            })
+          )}
+        >
+          {paragraph.sumAtAGlanceHeadline && <H2 className="font-light">{paragraph.sumAtAGlanceHeadline}</H2>}
```

Comment:
Why'd you move the heading to the bottom of the component? this will make all the content above not "below" the heading.

Reply:
Kat would like the content to stack on mobile in this order: Apply, Register, Receive, Attend, and How it works. She's like us to maintain the type-sizing as outlined in Figma when modifying the headings.

Here's what she's put together in Figma:
![image](https://github.com/user-attachments/assets/35b12f3d-68f1-448e-bce0-f7139134f1ee)

Thoughts @pookmish ?

Reply:
That's not good for heading structure. We can't build it that way. On mobile, the heading needs to go above.

Reply:
Besides, as a user, why would i see "How it works" _after_ the steps about how it works?